README.asc

```diff
@@ -11,10 +11,7 @@ For more information, see http://cassandra.apache.org/[the Apache Cassandra web
 
 Issues should be reported on https://issues.apache.org/jira/projects/CASSANDRA/issues/[The Cassandra Jira].
 
-Requirements
-------------
-. Java >= 1.8 (OpenJDK and Oracle JVMS have been tested)
-. Python 3.6+ (for cqlsh)
+
```

Comment:
Remove Requirements entirely or should they be replaced w/new requirements?

Reply:
this is just a readme file, which will always fall out of date.

this information can be found in other places (where it is more likely to be correct)

Reply:
Could we just link from the README to the related files where things will be more likely to be correct rather than just having inconsistent freshness on things here?

Reply:
fixed